test/functionalTest/cases/0725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -101,10 +102,11 @@ Date: REGEX(.*)
 =====================================================
 1
 0
+ftest
 --TEARDOWN--
```

Comment:
This may seem a little schizophrenic, but I have seen this test fali a few times, and always by finding TWO databases using the first command:

    echo "show dbs" | mongo | grep ${CB_DATABASE_NAME} | wc -l

This extra check will show us the exact name of the 'extra', unwanted database, and perhaps we can fix this sporadic failure thanks to this added check.